projects/commudle-admin/src/app/feature-modules/community-builds/components/create-community-build/create-community-build.component.html

```diff
@@ -64,6 +64,22 @@ <h1>
           </span>
         </div>
 
+        <div class="field teammates">
+          <p>Add teammate</p>
+          <div class="list" *ngFor="let user of emails; let in=index">
+            <div class = "input-box d-flex justify-content-space-between align-items-center">
```

Comment:
input-box??
better classname could be teammate-email-field, or just teammate-email